com.github.aggalex.Wineglass.yaml

```
@@ -0,0 +1,167 @@
+app-id: com.github.aggalex.Wineglass
```

```suggestion
app-id: com.github.aggalex.wineglass
```

given the rename that you shouldn't need.

---

oh wait, it MUST be `io.github`.

---

Unfortunately upstream is not active anymore. So i am not sure is it possible to fix any of this upstream. As for `io.github` i don't see if upstream mentioned it anywhere. So what exactly we should pick here for proper RDNS AppID and why please explain a little bit?

---

`io.github` has nothing to do with upstream and all to do with flaptak

https://docs.flatpak.org/en/latest/conventions.html#application-ids

---

Thanks!

---

@hfiguiere i reverted to `com.github.aggalex.Wineglass`. Because app won't start if we change this without upstream (no longer active). Here is log session bus:

```
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C2: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C3: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.io.github.aggalex.Wineglass
C4: -> org.freedesktop.DBus fake wildcarded AddMatch for io.github.aggalex.Wineglass
C5: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C2
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C3
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C4
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C5
C6: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C7: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C8: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C9: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C10: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C6
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C7
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C8
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C9
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C10
*SKIPPED*
C11: -> org.freedesktop.DBus call org.freedesktop.DBus.RequestName at /org/freedesktop/DBus
Filtering message due to arg0 com.github.aggalex.Wineglass, policy: 0 (required 3)
*HIDDEN* (ping)
B-1: <- org.freedesktop.DBus return from C11
*REWRITTEN*
Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```

---

no. you have to patch it. the linter will not allow it.

---

Ah, the linter now... Understood, thanks.